dist/api_err.py

```diff
@@ -75,7 +75,7 @@ def __init__(self, name, value, desc, long_desc=None, **flags):
 ]
 
 # Update the #defines in the wiredtiger.in file.
-tmp_file = '__tmp'
+tmp_file = '__tmp_err' + str(os.getpid())
```

Comment:
We could use the full name here too. Up to you.

Reply:
Done